cmd/wio/constants/project-type.go

```diff
@@ -1,6 +1,7 @@
 package constants
 
 const (
+    WIO = "wio"
```

Comment:
if you're refactoring can you make these not all caps? That's the go style apparently

Reply:
so it should be Wio?